react/content-list.jsx

```diff
@@ -17,11 +17,18 @@ let ContentList = React.createClass({
     }
   },
 
-  setStateFromDattCore: function () {
+  setStateFromDattCore: function (skipSetup) {
```

Comment:
Agree. However, I still think we should just complete Chris' mockups before taking a step back and revising/iterating. 
This will keep us honest about our milestone and also provide usecase validation for the Datt core methods by having a UI to work with.